gazelle/python/configure.go

```diff
@@ -99,6 +100,8 @@ func (py *Configurer) Configure(c *config.Config, rel string, f *rule.File) {
 	}
 
 	gazelleManifestFilename := "gazelle_python.yaml"
+	// TODO: figure out how to keep this in sync with pythonconfig.go New *Config
+	defaultVisibilityFmtString := "//%s:__subpackages__"
```

Comment:
Make this an exported `const` in the `pythonconfig.go`.

Reply:
Thanks, done.